### DISCUSSION TITLE
Make all unsaved and unassociated buffers their own CrapBook

### DISCUSSION THREAD

**Discussion by RolandHughes** [psemiletov/tea-qt#47] [Q&A]
All,

Is there a setting to make all unsaved and un-associated with a file buffers saved like CrapBook?

Sublime wants me to buy a new license. For years I have been using my licensed version of Sublime as my partial brain. When I'm researching things I'm cutting and pasting into shiny new buffers that aren't associated with files. If I ever actually get done with whatever those buffers related to, I nuke the buffer without saving. I paid for and have been using Sublime for years because it automatically saves all of that stuff and when I reboot then start Sublime, all of that stuff is there.

After kicking the tires on Tea again (used it years ago when it was the "infamous" thing at the bottom. I forget the name but remember the creator called it "infamous".) I see CrapBook works exactly the way I want every buffer not associated with a file to work. Is there a hidden setting to do that?

Thanks,

**Comment by psemiletov** [comment]
No, TEA can't save content of the unsaved files, but, in addition to Crapbook, TEA has the auto-saved .notes files that can be  created for any file (File - Notes).
